iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -1496,20 +1467,19 @@ def outputs(self, graph: fx.Graph) -> list[fx.Node]:
 
     @property
     def index(self) -> list[dict[IndexSymbol, IndexSequence]]:
+        def get_index(node: fx.Node) -> dict[IndexSymbol, IndexSequence]:
+            custom = get_custom(node)
+            if isinstance(custom, MMA):
+                return custom.acc_index
```

Comment:
Resurfacing my question from slack here: why do we need 2 different fields `acc_index` and `index` and risk them getting out of sync /  ?

Reply:
index here contains the index of the LHS, RHS and ACC nodes altogether as a sympy piecewise and allows one to query the index of the LHS, RHS or ACC from a single source of truth by calling a "specialize_index" function on the index. The reason for constructing this "unified" index was that rather than propagate 3 separate indices to nodes in the graph, it was simpler to propagate a single index and have each node then specialize the index as required. I see the potential sources of error here so open to suggestions.